src/components/activity/FormSetDates.js

```diff
@@ -0,0 +1,64 @@
+import {useState} from 'react';
+
+function FormSetDates({addActivity}) {
```

Comment:
I would recommend to prefix your state uplifting callbacks with `on`, in this case for example `onAddActivity`.
From the perspective of the calling component this is more consistent.